httputil/httputil.go

```diff
@@ -0,0 +1,77 @@
+package httputil
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+
+	"github.com/bitrise-io/go-utils/log"
+)
+
+// ReadBodyAndRestoreReadCloser ...
+func ReadBodyAndRestoreReadCloser(body *io.ReadCloser) (string, error) {
+	buf, err := ioutil.ReadAll(*body)
+	*body = ioutil.NopCloser(bytes.NewBuffer(buf))
+
+	if err != nil {
+		return "", err
+	}
+
+	return string(buf), nil
+}
+
+func printHeaders(header http.Header) {
```

Comment:
`printRequest` does not print 'padding logs' like this, but leaves it to the caller. 

We should keep logging consistent .

I think it's generally better to print 'Header: ' in the caller.